web/src/lib/components/i18n/__test__/format-message.spec.ts

```diff
@@ -57,15 +57,15 @@ describe('FormatMessage component', () => {
       key: 'html' as Translations,
       values: { name: 'test' },
     });
-    expect(container.innerHTML).toBe('Hello <strong>test</strong>');
+    expect(container.innerHTML).toBe('<!----><!---->Hello <!----><!----><!----><strong>test</strong>');
```

Comment:
perhaps we should check the `textContent` instead of the `innerHTML` so that we're not checking for all the `<!---->` comments

Reply:
We could, bu then we'd also use the strong formatting. Idk if we care though

Reply:
yeah, I don't know either. looks like @michelheusschen wrote the tests, so perhaps he has some opinion

we could also do something like:
```
expect(container.innerHTML.replaceAll('<!---->', '')).toBe('Hello <strong>test</strong>');
```

Reply:
Maybe that's just the easiest, yeah